coderefs/alias_test.go

```diff
@@ -71,6 +72,14 @@ func Test_GenerateAliases(t *testing.T) {
 			},
 			want: map[string][]string{testFlagKey: slice("SOME_FLAG")},
 		},
+		{
+			name:  "file wildcard pattern",
```

Comment:
Nit: I wonder if the previous test `file pattern` should be renamed to `exact filename pattern` to distinguish it from the wildcard test?

Reply:
updated!